mods/ts/rules/nod-support.yaml

```diff
@@ -196,7 +196,10 @@ NASTLH:
 		PowerupSpeech: EnablePower
 		PowerdownSpeech: DisablePower
 		IndicatorPalette: mouse
+	GrantConditionOnDisabled:
```

Comment:
It didn't have `DisabledOverlay` before, so I added just this to avoid making functional changes such as adding an overlay.